beast/tools/helpers.py

```diff
@@ -355,12 +357,11 @@ def val_in_unit(varname, value, defaultunit):
     -------
     >>> r = 0.5
     >>> print(val_in_unit('r', r, 'degree'))
-    # UserWarning: Variable r does not have explicit units. Assuming `degree`
-    <Quantity(0.5, 'degree')>
+    0.5 degree
 
     >>> r = 0.5 * unit['degree']
     >>> print(val_in_unit('r', r, 'degree'))
-    <Quantity(0.5, 'degree')>
+    0.5 degree
     """
```

Comment:
Note the changes here.  I got this to work.  
Basically, the >>> lines are run and the result in on the next line.  I had to update as the result has to be exactly what is returned by the print statement.